### DISCUSSION TITLE
Filament::isServing returns false during testing

### DISCUSSION THREAD

**Discussion by phh** [filamentphp/filament#10698] [Help]
Simple test, inspired by [the docs](https://filamentphp.com/docs/3.x/tables/testing):

```
it('list posts', function () {
    actingAsAdmin();

    livewire(ListPosts::class)
        ->assertOk()
        ->assertCountTableRecords(0);
});
```

However it returns 403, as `Filament::isServing()` returns false ([inspired by this](https://v2.filamentphp.com/tricks/scoping-policies-for-filament-using-isserving)):

```
        if (Filament::isServing()) {
            return $user->can('accessposts');
        }

        return $user->can(Permission::AccessVouchers->value) && $user->posts()->isPartOfBrand($brand->id);
```

Any idea why `Filament::setServingStatus();` is not booted here? Or do I need to do it manually?

**Comment by leandrocfe** [comment]
Did you implement the `FilamentUser` interface?

https://filamentphp.com/docs/3.x/panels/installation#allowing-users-to-access-a-panel

**Reply by phh** [reply]
Yes

`class User extends Authenticatable implements FilamentUser`

**Reply by leandrocfe** [reply]
Should it `actingAsAdmin()` access the list posts, right?

**Reply by phh** [reply]
Yes, that creates a user factory with the correct permissions.

That works, only thing that fails is that `Filament::isServing()` is returning false.
